packages/datetime/test/dateInputTests.tsx

```diff
@@ -14,6 +14,14 @@ import { padWithZeroes } from "../src/common/utils";
 import { Classes, DateInput } from "../src/index";
 
 describe("<DateInput>", () => {
+    it("handles null inputs without crashing", () => {
+        mount(<DateInput value={null} />);
```

Comment:
ok fair, i guess `assert.doesNotThrow` lets you validate the type of exception but we care about _any_ exception. this is reasonable.